packages/x-data-grid-pro/src/components/GridRowReorderCell.tsx

```diff
@@ -77,11 +78,34 @@ function GridRowReorderCell(params: GridRenderCellParams) {
     [apiRef, params.id],
   );
 
+  const onMouseDown = React.useCallback(() => {
```

Comment:
```suggestion
  const handleMouseDown = React.useCallback(() => {
```

Nit: use `handle` for event handler function names